File: DesktopApp/Model/Faculty.cs
using Newtonsoft.Json;

namespace ProfessorPerformanceEvaluation.Model
{
    public class Faculty
    {
        [JsonProperty("idFaculty")]
        public int IdFaculty { get; set; }
        [JsonProperty("name")]
        public string Name { get; set; }
    }
}

File: MobileApp/app/src/main/java/com/example/professorperformanceevaluation/model/Faculty.java
package com.example.professorperformanceevaluation.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Faculty {

    @SerializedName("idFaculty")
    @Expose
    private int idFaculty;
    @SerializedName("name")
    @Expose
    private String name;

    public Faculty() {
    }

    public int getIdFaculty() {
        return idFaculty;
    }

    public void setIdFaculty(int idFaculty) {
        this.idFaculty = idFaculty;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}